stdlib/public/core/Duration.swift

```diff
@@ -0,0 +1,228 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, macCatalyst 9999, *)
+@frozen
+public struct Duration: Sendable {
+  @frozen
+  @usableFromInline
+  struct Bits: Sendable, Codable, Hashable, Equatable {
```

Comment:
The Leading Underscore Rule is especially important to follow in `@inlinable`/`@usableFromInline` declarations, as  such names become ABI and therefore they pollute the public namespace. (E.g., if we went with `Bits` here, then we wouldn't be able to use this name for new public API in the future.)

```suggestion
  internal struct _Bits: Sendable, Codable, Hashable, Equatable {
```

Luckily, having the underscore in the type name means we won't need to underscore any of its members.